src/modules/launcher/Plugins/Microsoft.Plugin.Program/Main.cs

```diff
@@ -118,6 +104,20 @@ public void Init(PluginInitContext context)
             _context.API.ThemeChanged += OnThemeChanged;
 
             UpdateUWPIconPath(_context.API.GetCurrentTheme());
+
+            var a = Task.Run(() =>
```

Comment:
Moved from Constructor to Initialize method so it won't run if you disable the program plugin